pyface/tasks/tests/test_dock_pane_toggle_group.py

```diff
@@ -19,7 +19,7 @@
 from traits.etsconfig.api import ETSConfig
 
 
-USING_WX = ETSConfig.toolkit not in ["", "qt4"]
+USING_WX = ETSConfig.toolkit not in {"", "qt", "qt4"}
```

Comment:
This is for a separate issue/PR since the issue predates this PR, but this seems a bit odd - we're implicitly assuming that if it's not Qt (and it's not null), then it must be wx.

Reply:
Opened #1224